filament/backend/src/metal/MetalHandles.mm

```diff
@@ -372,7 +372,25 @@ BufferDescriptor b(nullptr, 0u, [](void* buffer, size_t size, void* user) {
     // All stages of the program have compiled successfuly, this is a valid program.
     isValid = true;
 
-    samplerGroupInfo = program.getSamplerGroupInfo();
+    auto& samplerGroupInfo = program.getSamplerGroupInfo();
```

Comment:
Please add a comment explaining what this code does and mention it needs to match the renaming scheme inside GLSLPostProcessor.cpp.

Reply:
Comments are added. Please let me know if there is missing.